superdesk/media/image.py

```diff
@@ -44,8 +56,13 @@ def get_meta(file_stream):
                 value = {ExifTags.GPSTAGS[vk].strip(): vv for vk, vv in v.items()}
                 exif_meta[key] = value
             else:
-                value = v.decode('UTF-8') if isinstance(v, bytes) else v
-                exif_meta[key] = value
```

Comment:
I don't think is a good placement here, it should be in a different method and also not a `getter` method.